101-bundle-json.md

```diff
@@ -453,5 +463,54 @@ The built-in actions (`install`, `upgrade`, `uninstall`) MUST NOT appear in the
 
 Implementations that do not support custom actions MUST NOT emit errors (either runtime or validation) if a bundle defines custom actions. That is, even if an implementation cannot execute custom actions, it MUST NOT fail to operate on bundles that declare custom actions.
 
+## Extensions
+
+In many cases, the bundle descriptor is a sufficient artifact for delivering a CNAB bundle, since invocation images and other images may be retrieved from registries and repositories. Consequently, it is important to provide an extension mechanism. An _extension_ is a named collection of auxiliary data whose meaning is defined outside of this specification.
+
+Tools MAY define and declare additional fields inside of the `extensions` section. Tools MUST NOT define additional fields anywhere else in the bundle descriptor. Implementations MAY produce an error or MAY ignore additional fields outside of the extension. However, implementations SHOULD preserve the data inside of the `extensions` section even when that information is not understood by the implementation.
+
+The `extensions` 
+
+```json
+{
+    "extensions": {
+        "example.com/duffle-bag": {
+            "icon": "https://example.com/icon.png",
+            "iconType": "PNG"
+        },
+        "example.com/backup-preferences": {
+            "frequency": "daily"
+        }
+    }
+}
+```
+
+The format is:
+
+```json
+{
+    "extensions": {
+        "EXTENSION NAME": "ARBITRARY JSON DATA"
```

Comment:
This gives the impression that `"ARBITRARY JSON DATA"` must be double quote delimited. Is the intention that this should be a JSON-encoded string or can it be arbitrary "inline" JSON?

Reply:
I guess the examples resolve that question, but best to have some normative text too.

Reply:
If only JSON had comments...

Reply:
No one say YAML. Please.